app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -1,19 +1,16 @@
 .gem-c-feedback {
   background: govuk-colour("white");
-  margin: 0 auto;
+  margin: govuk-spacing(6) auto 0 auto;
   max-width: $govuk-page-width;
   position: relative;
 
-  .visually-hidden {
-    @include govuk-visually-hidden;
-  }
-}
-
-.gem-c-feedback--top-margin {
-  margin-top: govuk-spacing(6);
-
   @include govuk-media-query($from: tablet) {
     margin-top: govuk-spacing(9);
+    border-bottom: 1px solid govuk-colour("white");
+  }
+
+  .visually-hidden {
+    @include govuk-visually-hidden;
```

Comment:
This isn't your change (in fact it's probably mine! but I can't remember) but do you know why the include is here, rather than using the class directly in the component? Seems like something worth fixing.